Gruntfile.js

```diff
@@ -388,7 +385,7 @@ module.exports = function(grunt) {
   var updateQuickTestsJSON = function() {
     var qtJSON = [];
     var rawtests = grunt.file.expand("quicktests/overlaying/tests/**/*.js");
-    rawtests.forEach(function(value, index, array){
+    rawtests.forEach(function(value){
```

Comment:
Was this change needed?
The default setting in eslint is `"args": "after-used"`, so it should not be the case to flag this one. Also, I personally prefer leaving in all arguments for a function, in case future development needs them